### PR TITLE
Internet

### PR DESCRIPTION
**Pull request by RachelRebecca**

### CODE REVIEW

File: src/main/java/weather/GetCurrentWeather.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+import java.io.*;
+
+public class GetCurrentWeather
+{
+    private OpenWeatherMapService service;
+    public GetCurrentWeather()
+    {
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl("http://samples.openweathermap.org")
```
Comment (andrewoid) [inline src/main/java/weather/GetCurrentWeather.java]: Use api.openweathermap.org

File: src/main/java/weather/CurrentWeatherFrame.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        catch (IOException e)
+        {
+            e.printStackTrace();
+        }
+    }
+
+    public void onNext(CurrentWeather currentWeather)
+    {
+        /*
+        new Consumer<CurrentWeather>()
```
Comment (andrewoid) [inline src/main/java/weather/CurrentWeatherFrame.java]: Don't keep commented out code

Comment (RachelRebecca) [reply src/main/java/weather/CurrentWeatherFrame.java]: I know I shouldn't, but since these are my class notes I was keeping them here for reference

Comment (RachelRebecca) [reply src/main/java/weather/CurrentWeatherFrame.java]: So that I don't have to keep coming onto GitHub's website to find them

**Review by RachelRebecca — COMMENTED** [review]
But I don't think it's giving back the right information

File: src/main/java/weather/CurrentWeatherJSONComment.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
+package weather;
+
+public class CurrentWeatherJSONComment
```
Comment (andrewoid) [inline src/main/java/weather/CurrentWeatherJSONComment.java]: You don't need to put this as a .java class you can put this in a .txt or better yet as a .json

**Pull request merged**